shell/src/main/java/alluxio/cli/LogLevel.java

```diff
@@ -192,6 +236,7 @@ private static void setLogLevel(final TargetInfo targetInfo, String logName, Str
     if (level != null) {
       uriBuilder.addParameter(LEVEL_OPTION_NAME, level);
     }
+    LOG.info("Setting log level on %s%n", uriBuilder.toString());
```

Comment:
Did you want this for log, or for System.out?

Reply:
I think users need not know the exact URL unless something goes wrong, then they can find it in the log. So I'd like this to be in the log.